resources/views/campaigns/forms/panes/ui.blade.php

```diff
@@ -121,6 +121,18 @@
             </div>
         </div>
 
+        <div class="col-md-6 col-lg-4">
+            <div class="form-group">
+                <label>
+                    {{ __('campaigns.ui.fields.connections_mode') }}
+                    <i class="fa-solid fa-question-circle hidden-xs hidden-sm" data-toggle="tooltip" title="{{ __('campaigns.ui.helpers.connections_mode') }}"></i>
+
+                </label>
+                {!! Form::select('ui_settings[connections_mode]', [0 => __('campaigns.ui.connections.default'), 1 => __('campaigns.ui.connections.direct'), 2 => __('campaigns.ui.connections.related'), 3 => __('campaigns.ui.connections.mentions')], null, ['class' => 'form-control']) !!}
+                <p class="help-block visible-xs visible-sm">{{ __('campaigns.ui.helpers.connections') }}</p>
```

Comment:
This second block is for mobile view, and it doest seem to use the same translation key than the tooltip. It should be the same key, since on mobile we don't have tooltips